src/main/java/org/fisco/bcos/web3j/protocol/core/methods/status/GenerateGroupStatus.java

```diff
@@ -0,0 +1,17 @@
+package org.fisco.bcos.web3j.protocol.core.methods.status;
+
+/** return status for rpc interface: generateGroup */
+public class GenerateGroupStatus {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'GenerateGroupStatus' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/app/fisco/web3sdk/pullRequest?prid=4732356)